laplace/curvature/asdl.py

```diff
@@ -15,13 +15,12 @@
 class AsdlInterface(CurvatureInterface):
     """Interface for asdfghjkl backend.
     """
-    def __init__(self, model, likelihood, last_layer=False):
+    def __init__(self, model, likelihood, last_layer=False, subnetwork_indices=None):
         if likelihood != 'classification':
             raise ValueError('This backend only supports classification currently.')
-        super().__init__(model, likelihood, last_layer)
+        super().__init__(model, likelihood, last_layer, subnetwork_indices)
 
-    @staticmethod
-    def jacobians(model, x):
+    def jacobians(self, model, x):
```

Comment:
If jacobians become an instancemethod, they should have the same interface as `gradients` because `self` has access to model. The model does not need to be passed. This requires plenty of change all over the library, I can take care of it later.

Reply:
Ah yeah, good point -- I'm also happy to make that change.

Reply:
Done.